src/test/scala/edu/ie3/mobsim/model/ElectricVehicleSpec.scala

```diff
@@ -44,38 +49,36 @@ class ElectricVehicleSpec extends UnitSpec with ElectricVehicleTestData {
                 homePoi,
                 workPoi,
                 storedEnergy,
-                destinationPoiType,
-                destinationCategoricalLocation,
                 destinationPoi,
                 parkingTimeStart,
                 departureTime,
                 chargingAtHomePossible,
                 chosenChargingStation,
                 chargingAtSimona,
-                finalDestinationPoiType,
                 finalDestinationPoi,
                 remainingDistanceAfterChargingHub,
                 chargingPricesMemory
               ) =>
             simulationStart shouldBe givenSimulationStart
+            uuid shouldBe ev.getUuid
             id shouldBe "test_car"
             model shouldBe "cool_producer cool_model"
             batteryCapacity shouldBe givenModel.capacity
             acChargingPower shouldBe givenModel.acPower
             dcChargingPower shouldBe givenModel.dcPower
             consumption shouldBe givenModel.consumption
-            homePoi shouldBe givenHomePoi
-            workPoi shouldBe givenWorkPoi
+            homePoi shouldBe ev.homePoi
+            workPoi shouldBe ev.workPoi
             storedEnergy shouldBe givenModel.capacity
             chargingAtSimona shouldBe false
-            destinationPoiType shouldBe PoiTypeDictionary.HOME
-            destinationCategoricalLocation shouldBe CategoricalLocationDictionary.HOME
+            ev.destinationPoi.getPoiType shouldBe PoiTypeDictionary.HOME
+            ev.destinationPoi.categoricalLocation shouldBe CategoricalLocationDictionary.HOME
             destinationPoi shouldBe givenHomePoi
```

Comment:
These two lines are obsolete, as `destinationPoi` is already checked in the next line